### PR TITLE
Changes the vault prefab to be a learn to hack prefab + overall nerfs it.

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
This PR changes the 3 gold bullions to 3 gold nuggets, and also reduces the amount of uqill bars to 1. The door to the prefab starts bolted and is unable to shock people. A multitool is added outside of the prefab to allow people to hack the door, plus writing on the floor guides people as to how to hack the floor. This is intended to serve as a way to teach players how to hack doors. 

Oh also the walls and doors are perspectivized I guess.

image

## Why's this needed? 
Currently this prefabs only purpose is easy powergaming, and is overall bland and generic. This should nerf its powergamingness, and also should make it fill a unique niche.

## Changelog

```changelog
(u)Ikea
(+)The vault prefab has had its contents nerfed, and has undergone minor changes.
```

### CODE REVIEW

**Comment by Conchuckter** [comment]
Couple things, I don't really see why having bullions is that big of an issue. Also the gold nuggets look really off compared to the bars, I know this would make them worthless but it might be better to make them normal gold bars.

Another idea is just to have 1 bullion instead of three.

**Comment by ThatFiveGuys** [comment]
Main thing is the economy is in a bad state to begin with, and this prefab is a  major reason why. This prefab is both easily identifable, easy to access, and contains over 100k thats easily gotten within 5 minutes. Its super easy to powergame the economy with.

**Comment by Conchuckter** [comment]
You make a good point and seems like this is the best option.

File: assets/maps/prefabs/prefab_vault.dmm [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+"c" = (/turf/simulated/wall/auto/reinforced/supernorn,/area/prefab/vault)
+"d" = (/obj/rack,/obj/item/raw_material/gold,/obj/item/raw_material/gold,/obj/item/raw_material/gold,/turf/simulated/floor{icon_state = "damaged2"},/area/prefab/vault)
+"e" = (/obj/rack,/obj/item/material_piece/uqill,/obj/machinery/light/incandescent/netural{dir = 1; pixel_y = 20},/turf/simulated/floor/void,/area/prefab/vault)
+"f" = (/obj/rack,/obj/item/material_piece/koshmarite,/obj/item/material_piece/koshmarite,/obj/item/material_piece/koshmarite,/turf/simulated/floor/void,/area/prefab/vault)
+"g" = (/turf/simulated/floor{icon_state = "damaged2"},/area/prefab/vault)
+"h" = (/turf/simulated/floor/void,/area/prefab/vault)
+"j" = (/obj/machinery/door/airlock/pyro/safe{icon_state = "generic_locked"; locked = 1},/turf/simulated/floor/scorched,/area/prefab/vault)
+"k" = (/obj/decal/cleanable/writing/infrared{desc = "The handwriting seems nearly human."; pixel_x = -1; pixel_y = 31; words = "The Siren screams its song, the fool joins it and slowly sinks into its domain. Only the wise ignore there predecessors and continue sailing without another thought."},/turf/variableTurf/floor,/area/prefab/vault)
```
Comment (ZeWaka) [inline assets/maps/prefabs/prefab_vault.dmm:9]: ?

Comment (ThatFiveGuys) [reply assets/maps/prefabs/prefab_vault.dmm:9]: If this is refering to the text, the basic idea of this is the vault is being used as a trap and lure by the eldritch to do ??? To people. That text was presumably written by someone effected by this trap, basically warning people to not fall for the trap.

Comment (ZeWaka) [reply assets/maps/prefabs/prefab_vault.dmm:9]: ok.
```suggestion
"k" = (/obj/decal/cleanable/writing/infrared{desc = "The handwriting seems nearly human."; pixel_x = -1; pixel_y = 31; words = "The Siren screams its song, the fool joins it and slowly sinks into its domain. Only the wise ignore their predecessors and continue sailing without another thought."},/turf/variableTurf/floor,/area/prefab/vault)
```

**Review by Tarmunora — CHANGES_REQUESTED** [review]
Is it intentional that it only prevents silicons from shocking the door? Snipping/pulsing the wires will still do it with your current impl

**Comment by ThatFiveGuys** [comment]
It didnt electrify when i tested it, though i should probably test again.

**Comment by Tarmunora** [comment]
the doors very much can shock if hacked incorrectly.

**Comment by ThatFiveGuys** [comment]
Okay when testing this I mistaked the random shocks for it being the shock wire, so tldr am dumb, fixed the code now.

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**